src/lib/cht-session.ts

```diff
@@ -99,10 +104,13 @@ export default class ChtSession {
   
     const adminRoles = ['admin', '_admin'];
     const isAdmin = _.intersection(adminRoles, resp.data?.roles).length > 0;
-    return {
-      isAdmin,
-      facilityId: resp.data?.facility_id,
-    };
+    let facilityId;
+    if (typeof resp.data?.facility_id === 'string') {
+      facilityId = [resp.data.facility_id];
+    } else if (Array.isArray(resp.data?.facility_id)) {
+      facilityId = resp.data.facility_id;
+    }
```

Comment:
else? throw?

Reply:
I didn't add it as there's an already existing null-check and throw where this function is called